test/loaderTest.js

```diff
@@ -71,6 +71,11 @@ describe("loader", function() {
 			'module.exports = "<!-- comment --><h3 customattr=\\"\\">#{number} {customer}</h3><p>{title}</p><!-- comment --><img src=\" + require("./image.png") + \" />";'
 		);
 	});
+	it("should preserve escaped quotes", function() {
+		loader.call({}, '<script>{"json": "with \\"quotes\\" in value"}</script>').should.be.eql(
+			'module.exports = "<script>{\\\"json\\\": \\\"with \\\\\\\"quotes\\\\\\\" in value\\\"}</script>";'
```

Comment:
This only occurs for `{ key: "bla \"blub\" bla" }`, but doesn't on `{ key: "bla 'blub' bla" }` right ?

Reply:
Hi Michael,
sorry for the late reply, but yes: that is correct. I've updated my minimal test case with two more examples: https://github.com/benurb/html-loader-escape-bug